### PR TITLE
Update bootstrap version of SDK and Arcade.

### PR DESCRIPTION
**Pull request by crummel**

### CODE REVIEW

**Comment by crummel** [comment]
This is passing locally now.  OSX and Windows have some temp file issues that I'm fixing up but I think it's ready for review.

File: global.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "msbuild-sdks": {
     "Microsoft.Build.CentralPackageVersions": "2.0.1",
     "Microsoft.NET.Sdk.IL": "3.0.0-preview-27107-01",
-    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19270.2"
+    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19306.12",
+    "Microsoft.Build.Traversal": "2.0.2"
```
Comment (dseefeld) [inline global.json]: Is this adding a prebuilt?  Who's requiring this?

File: repos/coreclr.proj [hunk 2]
```diff
@@ -20,15 +23,21 @@
...
     <BuildCommand Condition="$(Platform.Contains('arm'))">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
 
     <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension)</CleanCommand>
     <OfficialBuildId>20190422.75</OfficialBuildId>
+
+    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+    <PackagesOutput>$(ProjectDirectory)bin/Product/$(TargetOS).$(Platform).$(Configuration)/.nuget/pkg</PackagesOutput>
   </PropertyGroup>
 
+<Target Name="DebugStuff" AfterTargets="Build"> <Message Importance="High" Text="package output dir: $(PackagesOutput)" /> </Target>
```
Comment (dseefeld) [inline repos/coreclr.proj]: Is this just for debugging or expected to be permanent?

Comment (crummel) [reply repos/coreclr.proj]: Debugging, removed.

File: global.json (hunk 1 above)
Comment (crummel) [reply global.json]: I think I found a different way around this one, giving it a try without that package.

**Pull request merged**